.github/workflows/check-build-test.yml

```diff
@@ -124,13 +129,13 @@ jobs:
       - name: Cache Coursier cache
         uses: coursier/cache-action@v6.4.0
 
-      - name: Set up JDK ${{ matrix.java-version }}
+      - name: Set up JDK ${{ matrix.jdkVersion }}
         uses: coursier/setup-action@v1.3.0
         with:
-          jvm: ${{ matrix.java-version }}
+          jvm: ${{ matrix.jvmName }}
 
-      - name: Run tests with Scala ${{ matrix.scala-version }} and Java ${{ matrix.java-version }}
-        run: sbt ${{ matrix.sbt-opts }} "test"
+      - name: sbt test
+        run: sbt ++${{ matrix.scalaVersion }} test ${{ matrix.extraOpts }}
```

Comment:
Note that if we have mixed cross Scala versions in the sbt projects this will not work (for Scala 3).

Reply:
Thank you. That was added after I started on this.